hello-app-iap/README.md

```diff
@@ -0,0 +1,84 @@
+# Hello Application with IAP example
+
+> **Note:** This application is a copy of [hello-app](../hello-app) sample.
+> See that directory for more details on this sample.
+
+In this demo, we open two ports on the hello-app container, 8081
+and 8082, for two services. For each service, we assign a path, and then 
+use Cloud IAP to configure different access for it. To serve HTTPS traffic, we 
+will create a Google managed SSL certificate for the endpoint, so make sure you are using a 1.12+
+cluster.
+
+## Build the image
```

Comment:
Ideally we shouldn't document it here. This repo is just for the code backing the documentation.

There's an open bug (b/37857731) to create a tutorial explaining this. 
Do you mind moving this to a cloud.google.com tutorial in GKE doc set?